.github/workflows/sync-docs-from-node.yml

```
@@ -128,776 +116,570 @@ jobs:
-          set -euo pipefail
-          SYNC_REPORT="${{ runner.temp }}/sync_report.md"
-          SOURCE_API_DEBUG="source-repo/${{ steps.params.outputs.api_debug_path }}"
-          DEST_API_DEBUG="pages/api-references/genlayer-node/debug"
-          API_DEBUG_REGEX="${{ steps.params.outputs.api_debug_regex }}"
-
-          echo "" >> $SYNC_REPORT
-          echo "## API Debug Methods Sync" >> $SYNC_REPORT
-          printf "Using regex filter: \`%s\`\n" "$API_DEBUG_REGEX" >> $SYNC_REPORT
-          echo "" >> $SYNC_REPORT
+          echo "🔍 Checking if config sync report needs updating after sanitization"
           
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)